### PR TITLE
diphotons and bjets production

### PR DESCRIPTION
**Pull request by gourangakole**
Cards contain:
1) a a j j b (sherpa)
2) a a j b b (sherpa)

### CODE REVIEW

File: bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_gg_jbb_Mgg80-13000 [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#  Scales VAR{Abs2(p[2]+p[3])};
+#  ME_Generator Internal;
+#  Loop_Generator gg_yy;
+#  End process;
+
+  Process 93 93 -> 22 22 5 -5 93{1};
+  Order (*,2);
+  Enhance_Factor 2 {3};
+  Enhance_Factor 5 {4};
+  Enhance_Factor 10 {5};
```
Comment (qliphy) [inline bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_gg_jbb_Mgg80-13000]: Checking the Sherpa manual, especially on: https://sherpa.hepforge.org/doc/SHERPA-MC-2.2.4.html#Curly-brackets
Why do you have {3} {4} {5} here?   From the process definition,  you can only have additional jet,  so only 0 or 1-jet process?

File: bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_ggbjj_Mgg80-13000 [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  ME_QED Off;
+}(run)
+
+(processes){
+#  Process 21 21 -> 22 22;
+#  Scales VAR{Abs2(p[2]+p[3])};
+#  ME_Generator Internal;
+#  Loop_Generator gg_yy;
+#  End process;
+
```
Comment (qliphy) [inline bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_ggbjj_Mgg80-13000]: Please remove the above commented out lines

Comment (gourangakole) [reply bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_ggbjj_Mgg80-13000]: Sure I shall clean it up.

File: bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_gg_jbb_Mgg80-13000 (hunk 1 above)
Comment (gourangakole) [reply bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_gg_jbb_Mgg80-13000]: @qliphy Thanks for the link. 
I am not very sure about this. by looking at the old cards #1236 it was there. I would request if @pmillet and @michelif make any comment on this.

Comment (michelif) [reply bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_gg_jbb_Mgg80-13000]: don't know. If I remember, originally this datacard was created from ggjj, which had these enhance factors. https://github.com/cms-sw/genproductions/blob/master/python/ThirteenTeV/GGJets_M-60To200_Pt-50_13TeV-sherpa_cff.py#L44
So from that card I inferred that the number in brackets refers to the number of particles in the final state (including b-jets and gamma), but not an expert.

Comment (qliphy) [reply bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_gg_jbb_Mgg80-13000]: From ppewww.physics.gla.ac.uk/~ahgemmell/Sherpa/113Manual.pdf

You can find on slide 23 that "The number in curly brackets following a particle code specifies it’s maximal multiplicity" and an example there.  

I think in Gouranga's case "Process 93 93 -> 22 22 5 -5 93{1}"  should contain only subprocesses as aabb+0/1jets , and thus in the "Enhance_Factor n1 {n2}" lines,  "n2" should correspond to possible final multiplicity which are 4 or 5 here.

Comment (michelif) [reply bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_gg_jbb_Mgg80-13000]: yes, that's what i meant. i agree with removing " Enhance_Factor 2 {3};" if that's the issue (even if probably has zero effect)

Comment (qliphy) [reply bin/Sherpa/cards/production/2017/13TeV/higgs/Run.dat_13TeV_gg_jbb_Mgg80-13000]: Ok, then I suggest @gourangakole  to clean the cards to remove all parts on "{3}" but keep "{4}" and "{5}".

**Comment by gourangakole** [comment]
Thank you. could you please check once again @qliphy

**Comment by qliphy** [comment]
The cards Look good to me now.   @perrozzi @efeyazgan may have further checks before merging this.

**Pull request merged**